station-flattening-v0/src/main/java/net/modificationstation/stationapi/impl/level/DimensionStorage.java

```diff
@@ -0,0 +1,19 @@
+package net.modificationstation.stationapi.impl.level;
+
+import net.minecraft.util.io.CompoundTag;
+import net.modificationstation.stationapi.api.registry.Identifier;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class DimensionStorage {
```

Comment:
Is this used? The HashMap is never updated.

Reply:
No, at first I thought that I will store dimensional data separate from vanilla classes and I forgot to remove this thing

Reply:
I removed it